README.md

```diff
@@ -27,6 +29,17 @@ Metrics will now be accessible at http://localhost:8080/metrics
 
 To bind the java agent to a specific IP change the port number to `host:port`.
 
+__Note:__ Within the config.yaml...
+
+```
+hostPort: localhost:5555
+username: 
+password:
+```
+... is typically not required.
+
+### As an HTTP Server
+
 See `./run_sample_httpserver.sh` for a sample script that runs the httpserver against itself.
```

Comment:
Not directly related to this PR, but since we have a `jmx_example_application` in our integration tests, the `./run_sample_httpserver.sh` seems like a weird example. Maybe it's less confusing to exemplify both modes using the example application and to remove `./run_sample_httpserver.sh`:

Agent mode

```
java -javaagent:jmx_prometheus_javaagent/target/jmx_prometheus_javaagent-0.16.2-SNAPSHOT.jar=8080:example_configs/agent_sample_config.yml -jar integration_tests/jmx_example_application/target/jmx_example_application.jar
```

Standalone mode

```
java -javaagent:jmx_prometheus_javaagent/target/jmx_prometheus_javaagent-0.16.2-SNAPSHOT.jar=8080:example_configs/httpserver_sample_config.yml -jar integration_tests/jmx_example_application/target/jmx_example_application.jar

java -jar jmx_prometheus_httpserver/target/jmx_prometheus_httpserver-0.16.2-SNAPSHOT-jar-with-dependencies.jar 5556 example_configs/httpserver_sample_config.yml
```

What do you think?
```

Reply:
I agree, remove `./run_sample_httpserver.sh` and update the documentation.

It might make sense for your to merge my branch into your work so we can deliver it as a single commit?

Reply:
I could as well merge #675 right away and then we have your PR for the README.

Reply:
I merged it :)